node/src/components/consensus/protocols/highway/synchronizer.rs

```diff
@@ -21,6 +21,66 @@ use super::{HighwayMessage, ProtocolOutcomes, ACTION_ID_VERTEX};
 #[cfg(test)]
 mod tests;
 
+/// Incoming pre-validated vertices that we haven't added to the protocol state yet, and the
+/// timestamp when we received them.
+#[derive(DataSize, Debug)]
+pub(crate) struct PendingVertices<I, C>(HashMap<(I, PreValidatedVertex<C>), Timestamp>)
+where
+    C: Context;
+
+impl<I, C: Context> Default for PendingVertices<I, C> {
+    fn default() -> Self {
+        PendingVertices(Default::default())
+    }
+}
+
+impl<I: NodeIdT, C: Context> PendingVertices<I, C> {
+    /// Removes expired vertices and returns `true` if it is now empty.
+    fn remove_expired(&mut self, oldest: Timestamp) {
```

Comment:
The comment says _returns `true` if it is now empty_ but the method does not return anything.

Reply:
I don't know what `oldest` means in the context of this method. Can you give it a better name?